src/net/java/sip/communicator/impl/protocol/jabber/extensions/jingle/SourceRidGroupPacketExtension.java

```diff
@@ -0,0 +1,16 @@
+package net.java.sip.communicator.impl.protocol.jabber.extensions.jingle;
+
+/**
+ * Created by bbaldino on 7/6/17.
+ */
+public class SourceRidGroupPacketExtension extends SourceGroupPacketExtension
+{
+    /**
+     * The name of the "ssrc-group" element.
+     */
+    public static final String ELEMENT_NAME = "rid-group";
```

Comment:
Can we update the documentation to match what we discussed offline, i.e. that this maps to the `a=simulcast` SDP attribute. Also, s/ssrc-group/rid-group.